examples/lzapp-migration/tasks/common/taskHelper.ts

```diff
@@ -0,0 +1,941 @@
+import chalk from 'chalk'
```

Comment:
It almost feels as though this should live in `packages` in case more than one example should leverage the same functions?